src/cosmology/api/__init__.py

```diff
@@ -7,6 +7,15 @@
     CosmologyWrapperAPI,
     StandardCosmologyWrapperAPI,
 )
+from cosmology.api.components import (
```

Comment:
Wouldn't a relative import make more sense here?

Reply:
Normally yes, but absolute imports are both generally safer and play better with C-transpilation, e.g. mypyc. I tend to absolutify imports for those reasons.